From 0992848bc7ac30e39a41a63335e8550d70ae7d1f Mon Sep 17 00:00:00 2001
From: Martin Wilck <mwilck@suse.com>
Date: Wed, 13 Mar 2024 21:00:03 +0100
Subject: [PATCH] extensions/nbft: replace "nvme show-nbft" with "nvme nbft
 show"

Upstream nvme-cli has moved the nbft support into a plugin.
The syntax is now "nvme nbft show" rather than "nvme show-nbft".

Signed-off-by: Marius Tomaschewski <mt@suse.com>

diff --git a/extensions/nbft b/extensions/nbft
index 22af22760..6cb689892 100755
--- a/extensions/nbft
+++ b/extensions/nbft
@@ -252,7 +252,7 @@ nvme_show_nbft() {
 	if test "X$OPT_ROOTDIR" != "X" -a -f "$OPT_ROOTDIR/nbft.json" ; then
 		cat "$OPT_ROOTDIR/nbft.json"
 	else
-		nvme show-nbft -H -o json
+		nvme nbft show -H -o json
 	fi
 }
 
@@ -261,7 +261,7 @@ nbft_parse() {
 	local j=0 i
 
 	nbft_json=$(nvme_show_nbft 2>/dev/null) || {
-		warn "failed to read NBFT table, is \"nvme show-nbft\" supported?"
+		warn "failed to read NBFT table, is \"nvme nbft show\" supported?"
 		return $RC_ERROR
 	}
 	n_nbft=$(nbft_run_jq ". | length" "$nbft_json") || return 0
-- 
2.35.3

